packages/atomic/src/components/result-template-components/atomic-result-quickview/atomic-result-quickview.tsx

```diff
@@ -41,14 +41,17 @@ export class AtomicResultQuickview implements InitializableComponent {
 
   @ResultContext() private result!: Result;
 
+  private maximumPreviewSize?: number;
```

Comment:
Variable should be public and have the Prop decorator. It should also be documented & have a default value.

`@Prop() public maximumPreviewSize = 0;`